skunkbooth/main.py

```diff
@@ -43,6 +44,18 @@ def global_shortcuts(event: Event) -> None:
             raise StopApplication("User pressed quit")
 
 
+def saving_video(screen: Screen, vidBuf: Queue) -> None:
+    """Event handler to initiate video saving"""
+    effects = [
+        Cycle(screen, FigletText("SAVING", font="big"), screen.height // 2 - 8),
+        Cycle(screen, FigletText("VIDEO!", font="big"), screen.height // 2 + 3),
+        Stars(screen, (screen.width + screen.height) // 2),
+    ]
+    screen.play([Scene(effects, 100)], repeat=False)
+    if vidBuf.empty():
+        screen.close()
```

Comment:
This call to `screen.close()` seems redundant with the `screen.close()` call right after where `saving_video` is called. Also semantically, a function titled `saving_video` probably shouldn't be closing the screen

Reply:
Will fix

Reply:
For now you can drop the commits related to this from the PR so I can squash and merge this PR, and this can go in a separate PR.